scripts/install_quantization_libs.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import os
+import re
+import subprocess
+import sys
+
+EXTERNAL_REPOS_DIR = "external_repos"
+
+
+def remove_torch_from_setup(setup_file_path):
+    """Remove any torch requirement from the setup.py file."""
+    with open(setup_file_path, "r") as file:
+        setup_content = file.read()
+
+    # Use a regular expression to remove any line containing "torch=="
+    setup_content = re.sub(r'"torch==[^\"]+",', "", setup_content)
+
+    # Write the modified content back to setup.py
+    with open(setup_file_path, "w") as file:
+        file.write(setup_content)
+
+
+def clone_or_pull_repo(repo_url, repo_location_path):
+    """Clone the repo if it doesn't exist; otherwise, pull the latest changes."""
+    if os.path.exists(repo_location_path):
+        print(f"Directory {repo_location_path} already exists. Pulling the latest changes.")
+        subprocess.run(f"cd {repo_location_path} && git pull", shell=True, check=True)
+    else:
+        repo_name = repo_location_path.split("/")[-1]
+        print(f"Cloning {repo_name} into {repo_location_path}")
+        subprocess.run(f"git clone {repo_url} {repo_location_path}", shell=True, check=True)
+
+
+def install_autogptq_from_source():
+    """Install the AutoGPTQ package from GitHub."""
+    print("Installing AutoGPTQ package.")
+    autogptq_repo_path = os.path.join(EXTERNAL_REPOS_DIR, "AutoGPTQ")
+
+    clone_or_pull_repo("https://github.com/PanQiWei/AutoGPTQ.git", autogptq_repo_path)
+
+    subprocess.run("pip install numpy gekko pandas", shell=True, check=True)
+
+    subprocess.run(
+        f"cd {autogptq_repo_path} && {sys.executable} -m pip install -vvv --no-build-isolation -e .",
```

Comment:
non editable mode since we delete them afterwards
```suggestion
        f"cd {autogptq_repo_path} && {sys.executable} -m pip install -vvv --no-build-isolation .",
```